components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIManagerConfiguration.java

```diff
@@ -107,6 +108,7 @@ public class APIManagerConfiguration {
     private JSONArray applicationAttributes = new JSONArray();
     private CacheInvalidationConfiguration cacheInvalidationConfiguration;
 
+
```

Comment:
Shall we get rid of unwanted new lines,

Reply:
Fixed with f9052eea1b85d6e35e35bfb0c5a8ba69304cba05